src/Maths/math.vector.ts

```diff
@@ -3579,6 +3579,28 @@ export class Quaternion {
         Quaternion.FromRotationMatrixToRef(rotMat, ref);
     }
 
+    /**
+     * Creates a new rotation value to orient an object to look towards the given forward direction, the up direction being oriented like "up".
+     * This function works in left handed mode
+     * @param forward defines the forward direction - Must be normalized and orthogonal to up.
+     * @param up defines the up vector for the entity - Must be normalized and orthogonal to forward.
+     * @returns A new quaternion oriented toward the specified forward and up.
+     */
+    public static FromLookDirectionLH(forward: DeepImmutable<Vector3>, up: DeepImmutable<Vector3>): Quaternion {

```

Comment:
Would be great to also have toRef functions for those two, and use a temp Matrix instead of creating a new one. You have TmpVector available for you (in the same file), you could use TmpVector[7]

Reply:
Done